dotnet/samples/Demos/OpenAIRealtime/Program.cs

```diff
@@ -346,7 +346,7 @@ private static IEnumerable<ConversationTool> ConvertFunctions(Kernel kernel)
 
             foreach (var metadata in functionsMetadata)
             {
-                var toolDefinition = metadata.ToOpenAIFunction().ToFunctionDefinition();
+                var toolDefinition = metadata.ToOpenAIFunction().ToFunctionDefinition(false);
```

Comment:
We have customers using the code from this sample so can we make this parameter optional with a default to false?

Reply:
see my comment on optional parameters on the function definition. I'm fine either way, it's simply to maintain compatibility.